modules/tinymce/src/plugins/nonbreaking/test/ts/webdriver/NonbreakingWrapTypingTest.ts

```diff
@@ -148,8 +148,8 @@ describe('webdriver.tinymce.plugins.nonbreaking.NonbreakingWrapTypingTest', () =
                 s.text(str.is(Unicode.nbsp))
               ]
             }),
-            s.text(str.is(isGecko ? Unicode.zeroWidth + 'test ' : Unicode.zeroWidth + 'test\u00a0'))
-          ].concat(isGecko ? [ s.element('br', {}) ] : [])
+            s.text(str.is(Unicode.zeroWidth + 'test\u00a0'))
```

Comment:
Is there a reason you've swapped `\u00a0` to `Unicode.nbsp` everywhere else except in this file? It's fine either way, it's just inconsistent 🤷